src/components/PriorityDescriptionModal.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <p-icon class="priority-description-modal__icon" v-bind="attrs" icon="QuestionMarkCircleIcon" solid @click.stop.prevent="show = true" />
```

Comment:
idk how aggressively we want to embrace this pattern, but there is undoubtably other places in the app that could use this pattern. What do you think of abstracting this into something like HelpIconModal that takes icon, title, and text description for modal?

Reply:
Should the description in `HelpIconModal` be a prop or a slot as well?

Reply:
I think I'll turn it into slot in case we want to add styling (bolding some words for example) in the description